### DISCUSSION TITLE
SystaComfort Update v2.20.1

### DISCUSSION THREAD

**Discussion by jos-webservices** [xgcssch/SystaComfortPrometheusExporter#18] [General]
Hallo zusammen,
ich hab aktuell die 2.16 auf meiner SystaComfort II laufen. Jetzt hab ich bei Paradigma ein Update auf v2.20.1 gefunden. Weiß jemand, was sich geändert hat und ob sich das Update "lohnt"?

Vielen Dank
Jochen

**Comment by S-Bil** [comment]
Hallo Jochen, 

es wurde paar Fehler korrigiert, ein für angeschlossenes Bedienteil und ein für Pelletskessel bei der Anlagen ohne Pufferspeicher. 
Netzwerktechnisch wurde nichts geändert. 

Grüße
Serge

**Reply by jos-webservices** [reply]
Hi Serge,
danke für die schnelle Rückmeldung. Weißt du zufällig auch über die SW für den Peleo Optima bescheid? Ich hab gesehen, dass es eine 3.10 und 4.00 gibt, bei mir läuft noch die 2.09. Wird damit dann die REST-API wie bei Ökofen eingeführt? Oder bringen mir die Updates auch nicht viel mehr als eine schicke neue UI?
(Sorry für Offtopic, aber ich wusste nicht wie ich dir direkt schreiben kann.)

Viele Grüße
Jochen

**Reply by S-Bil** [reply]
Hallo Jochen, 

beim Kessel-Update wird komplizierter, nach dem Update muss man den Kessel einlernen, in der Regel macht es Heizungsbauer. Ich bin mir nicht sicher ob es der Sprung von 2.09 auf 4.01 nicht zu groß wäre, vielleicht muss man 3er Version zwischenspielen. 
Lieber doch über den Heizungsmonteur machen, vielleicht bei der nächsten Wartung. 

Grüße 
Serge

**Reply by jos-webservices** [reply]
Hi Serge,
ich würde das Update auf jeden Fall vom Heizungsbauer bei der nächsten Wartung machen lassen. Aber da der sicherlich die Zeit abrechnen wird, hätte mich davor interessiert ob sich das lohnt - oder nur die Optik anders wird. Wenn danach die REST-API verfügbar wäre, dann wäre es natürlich schon ne coole Sache.

Grüße
Jochen

**Reply by S-Bil** [reply]
Hallo Jochen,

Die Schnittstelle ist natürlich gleich geblieben, sollte eigentlich die Berechnung von Pelletsverbrauch angepasst sein und die Optik wurde komplett verändert. 

Grüße 
Serge

**Comment by bernhardguenter** [comment]
Hi,
Ich habe bzgl. Peleo Optima jetzt auch eine Frage. Gibt es eine Möglichkeit auf die Peleo Optima API zuzugreifen?
Ich sehe immer nur die Anmeldeseite ÖkeFen wofür ich keine Credentials habe. Hatte da auch schon mal erfolglos bei Paradigma angefragt.

Eine zweite Frage habe ich auch noch. Ich habe mir den Exporteur aufgesetzt und er schnurrt wie ein Kätzchen. Danke für die Arbeit!
Die Anzahl der Kesselstarts (systacomfortBoilerStartsTotal) wurde bei mir initial beim Aufsetzen des Exportes richtig gesetzt, allerdings ist der Wert dann gleich gleichgeblieben und ist nicht mehr mitgelaufen andere Werte passen zu der Wirklichkeit. Habe auch schon etwas debuggt aber nichts gefunden…

Hat jemand eine Idee?

Viele Grüße & besten Dank
Berni
